cmd/queue/main.go

```diff
@@ -498,16 +510,9 @@ func supportsMetrics(env config, logger *zap.SugaredLogger) bool {
 	return true
 }
 
-func buildAdminServer(healthState *health.State, probe *readiness.Probe, logger *zap.SugaredLogger) *http.Server {
+func buildAdminServer(healthState *health.State, probe *readiness.Probe) *http.Server {
 	adminMux := http.NewServeMux()
-	adminMux.HandleFunc(requestQueueHealthPath, healthState.HealthHandler(func() bool {
-		if !probe.ProbeContainer() {
```

Comment:
Missed this the first time, but `probe` is also unused.

Reply:
Done.